src/components/card/style/index.scss

```diff
@@ -46,4 +48,56 @@ $card-prefix-cls: '#{$awesome-prefix}-card';
 	&-body {
 		padding: $card-padding-base;
 	}
+
+	&-actions {
+		margin: 0;
+		padding: 0;
+		list-style: none;
+		background: $card-actions-background;
+		border-top: $border-width-base $border-style-base $border-color-split;
+		@include clearfix;
+
+		& > li {
```

Comment:
I wonder if it is necessary to have `&>` syntax here. Same for `span` and `anticon` bellow 🤔